search.py

```diff
@@ -49,6 +50,44 @@ def dfs(graph):
     # If there is no path from source cell to goal cell than return the string below
     return "Failure: No Path"
 
+
+def bfs(graph):
+    #creates a queue the size of the board so we have enough space
+    dim = len(graph)
+    queue = Queue(maxsize=(dim**2))
```

Comment:
Instead of importing a queue you can use a list as a queue in python, see this link: https://docs.python.org/3.1/tutorial/datastructures.html

Reply:
you still have to import a module should i do it that way?